core/templates/services/math-interactions.service.ts

```diff
@@ -96,6 +96,66 @@ export class MathInteractionsService {
     return errorMessage;
   }
 
+  isParenRedundant(
```

Comment:
See above, I think this check is written in a more complex way than it needs to be.

Reply:
Updated the check to only catch multiple consecutive parens.